docs/index.rst

```diff
@@ -0,0 +1,174 @@
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+   :caption: API Reference
+
+   reference/fairstep
+   reference/fairworkflow
+   reference/prov
+
+`fairworkflows` python library
```

Comment:
This is exactly the same text as in README.md but than converted to .rst using https://cloudconvert.com/md-to-rst , so no need to read both.